test/benchmark.jl

```diff
@@ -6,7 +6,7 @@ using NNlib: conv
 
 include("brusselator.jl")
 
-function benchmark_brusselator(N::Integer, method=:tracer)
+function benchmark_brusselator(N::Integer, method=:tracer_bitset)
```

Comment:
we could directly give the set type instead of a `method` symbol, at least outside of symbolics

Reply:
We should just make use of the ADTypes interface here. CC #46.

Reply:
Opened an issue to keep track #48